Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Defining ruby version
+FROM ruby:2.6.6-alpine
+
+# Copy app and set working dir
+COPY . /application
+WORKDIR /application
+
+# Prerequisites for gems install
+RUN rm -rf /application/Gemfile.lock
+RUN apk add build-base \
+            npm \
+            tzdata \
+            git
+
+# Install bundler and gems
+RUN gem install bundler:2.1.4
+RUN bundle install --without="development"
+
+# Set environment variables and expose the running port
+ENV RELATIVE_URL_ROOT="/"
+ENV RAILS_ENV="production"
+EXPOSE 3000
+
+# Add secrets as environment variables (used development env key temporarily)
+ENV SECRET_KEY_BASE="c73c9b7ba9910c6f9a3bdd193392f8da6cc36094dbbd3f8c5e575cfc03b09e9ad0d056e15f591bc5f39adbc2e35398ec75cbac94cef5668f8a4497853e5d0537" 
```

Comment:
Have a look at the `entrypoint.sh` in [regulated-products](https://github.com/epimorphics/regulated-products-app/blob/dev/entrypoint.sh). You will also notice that the `SECRET_KEY_BASE` is loaded using `rails secret` command rather than being stored explictly like above. Generally try and avoid storing any secret keys in github repos (having said that I'm the first person to do just that every now and then :laughing:)